tests/sentry/lang/javascript/test_processor.py

```diff
@@ -94,6 +96,108 @@ def test_missing_dist(self, _1, _2):
         assert processor.dist.name == "foo"
         assert processor.dist.date_added.timestamp() == processor.data["timestamp"]
 
+    def test_tag_suspected_console_error(self):
+        project = self.create_project()
+        release = self.create_release(project=project, version="12.31.12")
+
+        data = {
+            "is_exception": True,
+            "platform": "javascript",
+            "project": project.id,
+            "exception": {
```

Comment:
I have verified that this almost have the same shape as [an event](https://sentry.io/organizations/sentry/issues/3644209404/events/f0be336835d5424ea2470c3130f09fae/?project=11276) I found [1]

I noticed differences in the frame:
- `"abs_path": "<anonymous>",`
- `"in_app": true`

Are these differences important to take into account?

[1]
```
 "exception": {
    "values": [
      {
        "type": "TypeError",
        "value": "Cannot read properties of undefined (reading 'toto')",
        "stacktrace": {
          "frames": [
            {
              "filename": "<anonymous>",
              "abs_path": "<anonymous>",
              "lineno": 1,
              "colno": 11,
              "in_app": true
            }
          ]
        },
        "mechanism": {
          "type": "onerror",
          "handled": false
        }
      }
    ]
  },
```

Reply:
That's a good question! I'll make a note of it and we can look into whether the existing logic tags events accurately. We can iterate on the logic if we're missing a subset of events.